src/components/accordion/index.md.njk

```diff
@@ -17,83 +17,119 @@ The accordion component lets users show and hide sections of related content on
 
-Do user research to find out the best order for the sections.
+#### Write clear button text
 
-### Use clear labels
+Write the heading and summary text like any other button text. Use sentence case, describe the content that will show, and keep it short.
+Users of screen readers might find it difficult to navigate the accordion if the button text is too long.
 
-Accordions hide content, so the labels need to be clear. If necessary, you can add a summary line to help users understand what is in the section.
+If you struggle to come up with clear button text, it might be because the way you’ve separated the content is not clear. Organise sections in a way that makes sense to users, based on their needs.
+
+#### Adding a summary line
+
+Only add a summary line if it’s actually needed, as it's likely to make the button text too long.
+
+If you’ve decided that you need the summary line, you must make it as short as possible.
 
 {{ example({group: "components", item: "accordion", example: "with-summary-section", html: true, nunjucks: true, open: false, size: "xl"}) }}
 
-If you struggle to come up with clear labels, it might be because the way you’ve separated the content is not clear.
+#### Structure section headings with the rest of the page
+
+The accordion component shows section headings as `<h2>` headings. If needed, change the heading level of the section headings to make them fit within the other headings on the page.
+
+### Change how an accordion starts
+
+Users might need some sections to be open from the start. If they leave and then return to the page, they might also need sections they opened to stay open.
+
+If the user leaves and then returns to the page, the accordion component will remember if sections were opened or closed. You can configure sections to start and stay open, but not stay closed.
 
 ### Do not disable sections
 
 Disabling controls is normally confusing for users. If there is no content for a section, either remove the section or, if this would be confusing for your users, explain why there is no content when the section is opened.
 
 ## Research on this component
 
-This component is experimental because there are different approaches to accordions in services. More research is needed to determine the best approach, or to know when a different approach works better.
+We updated this component in November 2021 to solve an accessibility issue where the buttons and section labels might be mistaken for links.
 
-The design for this component was originally created and tested by a team at the Government Digital Service (GDS) on the [GOV.UK Service Manual](https://www.gov.uk/service-manual) and [GOV.UK topic pages](https://designnotes.blog.gov.uk/2017/06/29/designing-new-navigation-elements-for-gov-uk/).
+[Read about the research and development that went into improving the accordion component](https://github.com/alphagov/govuk-design-system-backlog/issues/1).
```

Comment:
Deep link to v4.0.0 backlog post (when published)

Reply:
Thanks @calvin-lau-sig7 , I've added that as a checklist item at the bottom of the release plan so it doesn't block this PR being merged.
https://docs.google.com/document/d/1DGbfrjQE03Nm1OZORqGSMNrsu_BmBlvveAcWzCvNM-0/edit#